src/Servers/Kestrel/test/FunctionalTests/Http2/ShutdownTests.cs

```diff
@@ -100,7 +100,6 @@ public async Task GracefulShutdownWaitsForRequestsToFinish()
         }
 
         [ConditionalFact]
-        [QuarantinedTest]
         public async Task GracefulTurnsAbortiveIfRequestsDoNotFinish()
```

Comment:
I don't think this should be unquarantined. It looks like only the Libuv.FunctionalTests copy of this test meets the requirements.

Reply:
Ah, good point about how this should only be quarantined for the sockets test.

Reply:
Here too, I'd go back to unquarantining both tests. I looked more closely and the Sockets variant has (I think) also been successful for 30 days. But, ignore my follow up if I'm reading things wrong 😺

Reply:
I don't think so, it looks like GracefulTurnsAbortiveIfRequestsDoNotFinish has failed 7 times in the last 30 days https://whatsbroken.azurewebsites.net/quarantined. So I'd rather not unquarantine.